src/storage/storage.js

```diff
@@ -26,7 +14,57 @@ function obtenerPorId(idEquipo){
     return equipoSeleccionado
 }
 
+function guardarEquipo(nuevoEquipo){
+    const equipos = obtenerTodosLosEquipos()
+
+    equipos.push(nuevoEquipo)
+    fs.writeFileSync("./data/listaEquipos.json", JSON.stringify(equipos))
+
+    return dataMapper(nuevoEquipo)
+}
+
+function guardarCambiosEquipo(equipoEditado){
+    const equipos = obtenerTodosLosEquipos()
+
+    for(let i = 0; i < equipos.length; i++){
+        if(Number(equipoEditado.numeroId) === Number(equipos[i].numeroId)){
+            equipos.splice(i, 1, equipoEditado)
+        }
+    }
+
+    fs.writeFileSync("./data/listaEquipos.json", JSON.stringify(equipos))
+
+    return equipoEditado
+}
+
+function guardarBorrarEquipo(id){
+    const equipos = obtenerTodosLosEquipos()
+
+    for(let i = 0; i < equipos.length; i++){
+        if(Number(equipos[i].numeroId) === Number(id)){
+            equipos.splice(i, 1)
+            break
+        }
+    }
+
+    fs.writeFileSync("./data/listaEquipos.json", JSON.stringify(equipos), "utf-8")
+}
+
+function obtenerTodosLosEquipos(){
+    const clubes = JSON.parse(fs.readFileSync("./data/listaEquipos.json", "utf-8"))
+    const listaEquipos = clubes.map( club => {
+        return dataMapper(club)
+    })
```

Comment:
acordate que las arrow functions devuelven el valor por default, podés hacer

```suggestion
    const listaEquipos = clubes.map(club => dataMapper(club));
```